src/context/UserContext.tsx

```diff
@@ -6,6 +6,12 @@ export type AuthUser = {
   id: number,
   name: string,
   googleId: string,
+  buddies: Buddy[];
+};
+
+export type Buddy = {
+  id: number;
+  name: string;
```

Comment:
Does this have any weird effects on the user context? Is the current value of buddies in user context null?

Reply:
This doesn't have any weird effects but it is not being used right now. I commented it out right now until I get back around to using it or removing it if no longer needed.

Reply:
Sounds good!